### PR TITLE
Feature 1 : Add Routing to Application

### PR DESCRIPTION
**Pull request by Abhishek-Jain-1925**
Feature 1 : Add Routing to Application

Work done are as follows - 
1.Added Router file with specified some primary routes.
2.Added dummy templates where required.
3.Installation and Setup of libs.
4.Profile Listing template.

### CODE REVIEW

File: src/components/Login/login.js
Comment (Payal13) [inline src/components/Login/login.js:1]: fine name should be `.../Login/index.js`

File: src/components/Resume/ListProfiles.js
Comment (Payal13) [inline src/components/Resume/ListProfiles.js:1]: File name and path - `...components/Profile/List/index.js`

File: src/components/Resume/ListProfiles.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const ListProfiles = () => {
+  return (
+    <div>
```
Comment (Payal13) [inline src/components/Resume/ListProfiles.js]: use React Fragments instaed of `div` wherever we can

File: src/routes/router.jsx
Comment (Payal13) [inline src/routes/router.jsx:1]: use .js file

File: src/routes/router.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import Login from "../components/Login/login";
+import Dashboard from "../components/Dashboard";
+import { Editor } from "../components/Builder/Index";
+import ListProfiles from "../components/Resume/ListProfiles";
+
+const Router = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Login />} />
```
Comment (Payal13) [inline src/routes/router.jsx]: path names - 
`/login`
`/profiles`
`/profile-builder`

**Review by Payal13 — CHANGES_REQUESTED** [review]

File: src/routes/router.jsx (hunk 2 above)
Comment (Abhishek-Jain-1925) [reply src/routes/router.jsx]: Added suggested path names

File: src/components/Login/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Login = () => {
+  return (
+    <React.Fragment>
```
Comment (Payal13) [inline src/components/Login/index.js:5]: You can use like - 
```
<>
    <h1>Login Page will Appear Here!</h1>
</>
```

File: src/routes/router.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import { Editor } from "../components/Builder/Index";
+import ListProfiles from "../components/Resume/ListProfiles";
+
+const Router = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/login" element={<Login />} />
+        <Route path="/profiles" element={<ListProfiles />} />
+        <Route path="/dashboard" element={<Dashboard />} />
```
Comment (Payal13) [inline src/routes/router.js:14]: what is dashboard used for ?

**Review by Payal13 — APPROVED** [review]

File: src/routes/router.js (hunk 4 above)
Comment (Abhishek-Jain-1925) [reply src/routes/router.js:14]: HomePage Design for initial Rendering

**Pull request merged**